service/internal/builder/exporters_builder.go

```diff
@@ -125,7 +125,7 @@ func (exps Exporters) ToMapByDataType() map[config.DataType]map[config.Component
 
 type dataTypeRequirement struct {
 	// Pipeline that requires the data type.
```

Comment:
I'm not sure the comment makes sense with the change.

Reply:
Done, simplified this more.